samples/cpp/benchmark_app/main.cpp

```diff
@@ -988,60 +1006,21 @@ int main(int argc, char* argv[]) {
                 }
                 statistics->add_parameters(
                     StatisticsReport::Category::EXECUTION_RESULTS,
-                    {
-                        {latency_label, double_to_string(generalLatency.percentile(FLAGS_latency_percentile))},
-                    });
-                statistics->add_parameters(StatisticsReport::Category::EXECUTION_RESULTS,
-                                           {
-                                               {"Average latency (ms)", double_to_string(generalLatency.average())},
-                                           });
-                statistics->add_parameters(StatisticsReport::Category::EXECUTION_RESULTS,
-                                           {
-                                               {"Min latency (ms)", double_to_string(generalLatency.min())},
-                                           });
-                statistics->add_parameters(StatisticsReport::Category::EXECUTION_RESULTS,
-                                           {
-                                               {"Max latency (ms)", double_to_string(generalLatency.max())},
-                                           });
+                    {StatisticsVariant(latency_label, "latency_median", generalLatency.median_or_percentile),
+                     StatisticsVariant("Percentile boundary", "percentile_boundary", FLAGS_latency_percentile),
+                     StatisticsVariant("Average latency (ms)", "latency_avg", generalLatency.avg),
+                     StatisticsVariant("Min latency (ms)", "latency_min", generalLatency.min),
+                     StatisticsVariant("Max latency (ms)", "latency_max", generalLatency.max),
+                     StatisticsVariant("throughput", "throughput", fps)});
```

Comment:
```suggestion
                     StatisticsVariant("Average latency (ms)", "latency_avg_ms", generalLatency.avg),
                     StatisticsVariant("Min latency (ms)", "latency_min_ms", generalLatency.min),
                     StatisticsVariant("Max latency (ms)", "latency_max_ms", generalLatency.max),
                     StatisticsVariant("throughput", "throughput_fps", fps)});
```
I suggest adding measurement units for labels where needed

Reply:
No, that was a requirements from QA to get rid of them